backend/Program.cs

```diff
@@ -69,7 +71,10 @@ public static async Task Main(string[] args)
                 var userManager = scope.ServiceProvider.GetRequiredService<UserManager<User>>();
                 await SampleUsers.CreateAdminAsync(userManager);
                 await SampleUsers.CreateUsersAsync(userManager);
-                
+
+                var seeder = scope.ServiceProvider.GetService<SeedSightings>();
```

Comment:
can this be renamed to 'sightingsSeeder' inline with 'speciesSeeder' naming below